checks.d/kubernetes_topology.py

```diff
@@ -64,46 +67,73 @@ def _extract_nodes(self, kubeutil, instance_key):
         for node in kubeutil.retrieve_nodes_list()['items']:
             data = dict()
             addresses = {item['type']: item['address'] for item in node['status']['addresses']}
-            data['labels'] = self._flatten_dict(kubeutil.extract_metadata_labels(node['metadata']))
+            data['labels'] = self._make_labels(kubeutil, node['metadata'])
             data['internal_ip'] = addresses['InternalIP']
             data['legacy_host_ip'] = addresses['LegacyHostIP']
             data['external_ip'] = addresses['ExternalIP']
             data['hostname'] = addresses['Hostname']
             self.component(instance_key, node['metadata']['name'], {'name': 'KUBERNETES_NODE'}, data)
 
+    def _extract_deployments(self, kubeutil, instance_key):
```

Comment:
A comment with a link on what deployments are and the json spec would be good here